org.eclipse.scout.rt.shared.test/src/test/java/org/eclipse/scout/rt/shared/servicetunnel/http/HttpServiceTunnelTest.java

```diff
@@ -86,6 +85,11 @@ protected IHttpTransportManager getHttpTransportManager() {
               .setLowLevelHttpResponse(expectedResponse)
               .build();
 
+          @Override
+          public String getName() {
+            return "scout.test-service-tunnel";
```

Comment:
Maybe we should add a more specific domain.

e.g.
```suggestion
            return "scout.transport.test-service-tunnel";
```

or
```suggestion
            return "scout.http.test-service-tunnel";
```


More examples: `scout.transport.basic-auth`/`scout.http.basic-auth` and `scout.transport.default`/`scout.http.default`. I would favor `transport` over `http` as the corresponding attribute name (`http.client.name`) already contains `http` in its name.

Reply:
Change to:

- "scout.transport.basic-auth"
- "scout.transport.default"
- "scout.transport.service-tunnel"
- "scout.transport.test-service-tunnel"